### PR TITLE
sanctuary-scripts@7.0.x

### PR DESCRIPTION
**Pull request by Avaq**
This also updates the tests to work with Oletus as a runner.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -91,14 +103,13 @@ test ('createRunner', () => {
       resultLine,
       /^.*$/,
       /^$/,
-    ])
+    ], run)
   );
 
-  const T = '\u001b\\[90m│\u001b\\[39m';
+  const T = '│';
```
Comment (Avaq) [inline test/index.js]: This is the only surprising change to me. The table separators seem to no longer be rendered in grey.

Comment (Avaq) [reply test/index.js]: Ah, I see this change causes the tests to break on CI. So maybe it's the Node version?

Comment (Avaq) [reply test/index.js]: Nope. The tests pass for me locally on Node 18 too without the expectations of terminal colors. :thinking:

Comment (Avaq) [reply test/index.js]: No idea.. something on my local system is causing those ansii codes to be suppressed. Oh well. I've amended the commit to keep things how they were.

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   },
   "devDependencies": {
     "intercept-stdout": "0.1.x",
-    "sanctuary-scripts": "6.0.x",
+    "sanctuary-scripts": "^7.0.0",
```
Comment (davidchambers) [inline package.json]:
```suggestion
    "sanctuary-scripts": "7.0.x",
```

File: test/index.js [hunk 3]
```diff
@@ -157,63 +149,58 @@ test ('split runner syntax', done => {
...
-  const spec = {mock: [{}, () => {}]};
-  const run = createRunner ({}, {}, {}, spec);
-  const assert = expectOutput ([
+await test ('no significant difference', () => expectOutput (
+  [
     /^# 1[/]1: mock *\r$/,
     /^┌[─┬]+┐$/,
     /^│ suite +│ left +│ right +│ diff +│ change +│ α │$/,
     /^├[─┼]+┤$/,
+    // eslint-disable-next-line max-len
```
Comment (davidchambers) [inline test/index.js]: Instead, let's set `"max-len": ["off"]` for `test/**/*.js` in __.eslintrc.json__ as we do in other projects.

**Review by davidchambers — APPROVED** [review]

**Pull request merged**